### PR TITLE
Move tests to folders

### LINKED ISSUE #42: Move tests to tests folder to increase readability of commits/PRs

### PR DESCRIPTION
**Pull request by NinoLipartiia**
Close #42

### CODE REVIEW

**Comment by deveusss** [comment]
@NinoLipartiia There are some conflicting changes on this:)

**Comment by NinoLipartiia** [comment]
Fixed these conflicts

File: contracts/cw-croncat/src/tests/receiver.rs [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+
+pub fn contract_template() -> Box<dyn Contract<Empty>> {
+    let contract = ContractWrapper::new(
+        crate::entry::execute,
+        crate::entry::instantiate,
+        crate::entry::query,
+    )
+    .with_reply(crate::entry::reply);
+    Box::new(contract)
+}
```
Comment (Buckram123) [inline contracts/cw-croncat/src/tests/receiver.rs:23]: What do you think about moving contract_template() to `helpers.rs`?

File: contracts/cw-croncat/src/tests/receiver.rs [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+        //Must send some available balance for rewards
+        .instantiate_contract(
+            cw_template_id,
+            owner_addr.clone(),
+            &msg,
+            &coins(2_000_000, NATIVE_DENOM),
+            "Manager",
+            None,
+        )
+        .unwrap();
```
Comment (Buckram123) [inline contracts/cw-croncat/src/tests/receiver.rs:83]: Same about move to `helpers.rs`?

File: contracts/cw-croncat/src/tests/receiver.rs [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+        .instantiate_contract(cw20_id, owner_addr, &msg, &[], "Fungible-tokens", None)
+        .unwrap();
+    (app, cw_template_contract, cw20_addr)
+}
+
+pub fn add_little_time(block: &mut BlockInfo) {
+    // block.time = block.time.plus_seconds(360);
+    block.time = block.time.plus_seconds(19);
+    block.height += 1;
+}
```
Comment (Buckram123) [inline contracts/cw-croncat/src/tests/receiver.rs:109]: and this

File: contracts/cw-croncat/src/tests/tasks.rs [hunk 4]
```diff
@@ -0,0 +1,1140 @@
...
+        crate::entry::instantiate,
+        crate::entry::query,
+    );
+    Box::new(contract)
+}
+
+const ADMIN: &str = "cosmos1sjllsnramtg3ewxqwwrwjxfgc4n4ef9u0tvx7u";
+const ANYONE: &str = "cosmos1t5u0jfg3ljsjrh2m9e47d4ny2hea7eehxrzdgd";
+const VERY_RICH: &str = "cosmos1c3cy3wzzz3698ypklvh7shksvmefj69xhm89z2";
+const NATIVE_DENOM: &str = "atom";
```
Comment (Buckram123) [inline contracts/cw-croncat/src/tests/tasks.rs:32]: maybe accounts too

File: contracts/cw-croncat/src/tests/tasks.rs [hunk 5]
```diff
@@ -0,0 +1,1140 @@
...
+
+fn mock_app() -> App {
+    AppBuilder::new().build(|router, _, storage| {
+        let accounts: Vec<(u128, String)> = vec![
+            (100, ADMIN.to_string()),
+            (800_010, ANYONE.to_string()),
```
Comment (Buckram123) [inline contracts/cw-croncat/src/tests/tasks.rs:38]: and `mock_app()`

**Review by Buckram123 — APPROVED** [review]
Few fixes possibly for another issue, IMO these are not critical, so LGTM

File: contracts/cw-croncat/src/tests/receiver.rs (hunk 1 above)
Comment (TrevorJTClarke) [reply contracts/cw-croncat/src/tests/receiver.rs:23]: I like this idea

File: contracts/cw-croncat/src/tests/slots.rs [hunk 6]
```diff
@@ -0,0 +1,376 @@
...
+        println!("next_id {:?}, slot_kind {:?}", next_id, slot_kind);
+        assert_eq!(outcome_block, &next_id);
+        assert_eq!(outcome_slot_kind, &slot_kind);
+    }
+}
+
+#[test]
+fn interval_get_next_block_by_offset() {
+    // (input, input, outcome, outcome)
+    let cases: Vec<(Interval, BoundaryValidated, u64, SlotType)> = vec![
```
Comment (TrevorJTClarke) [inline contracts/cw-croncat/src/tests/slots.rs:104]: Could add skip fmt for these lines - to help with readability

File: packages/cw-croncat-core/src/tests/msg.rs [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+        TaskResponse,
+    },
+    types::{
+        Action, Agent, AgentResponse, AgentStatus, Boundary, BoundaryValidated, GenericBalance,
+        Interval, SlotType, Task,
+    },
+};
+
+#[test]
+fn everything_can_be_de_serealized() {
```
Comment (TrevorJTClarke) [inline packages/cw-croncat-core/src/tests/msg.rs:17]: `serealized` -> `serialized`

**Review by TrevorJTClarke — APPROVED** [review]
This is so needed! Thank you!

Optional items found, happy to have those happen on a future PR

**Review by TrevorJTClarke — APPROVED** [review]
Much more organized! thank you!

**Pull request merged**